inc/communicationnetworkdiscovery.class.php

```diff
@@ -103,45 +103,32 @@ function import($p_DEVICEID, $a_CONTENT, $arrayinventory) {
       }
 
       if ($pfTaskjobstate->getFromDB($a_CONTENT['PROCESSNUMBER'])) {
-         if ($pfTaskjobstate->fields['state'] != "3") {
+         if ($pfTaskjobstate->fields['state'] != PluginFusioninventoryTaskjobstate::FINISHED) {
             $pfImportExport = new PluginFusioninventorySnmpmodelImportExport();
             $errors .= $pfImportExport->import_netdiscovery($a_CONTENT, $p_DEVICEID);
             if (isset($a_CONTENT['AGENT']['END'])) {
-               if ((isset($a_CONTENT['DICO'])) AND ($a_CONTENT['DICO'] == "REQUEST")) {
```

Comment:
I really do not know how to be sure this is still used

Reply:
it was an old behavior of the agent:it had an auto update mechanism for network discovery mapping file. If the agent had a outdated version, the server would send a message to stop the process. Next time, the server would send the new file to the agent and the process could continue.
If was complicated and frustrating for the user, it has been removed a long time ago.

Reply:
OK, thank you for the explanations :)